### PR TITLE
fix(form-item): validation input or change and blur, The animation triggers twice

### PR DESCRIPTION
**Pull request by XueHua-s**
close  https://github.com/tusen-ai/naive-ui/issues/5583

### CODE REVIEW

**Comment by XueHua-s** [comment]
n-form表单错误信息动画优化
当form表单的rule设置了多个trigger，比如trigger: ['input', 'blur']，此时input会触发错误信息，blur也会触发错误信息，错误出现会有动画效果，导致页面错误信息展示再闪一次出现。
1、打开官网示例：https://www.naiveui.com/zh-CN/os-theme/components/form
2、在年龄输入框中输入任意值并点击其他地方让输入框失去焦点
3、再次点击输入框，将内容删除，此时会出现必填的校验报错，再点击其他地方让输入框失去焦点，此时会再次出现必填项的错误校验信息

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "naive-ui",
-  "version": "2.37.3",
+  "version": "2.37.4",
```
Comment (OrbisK) [inline package.json:3]: this should not be changed within a pr.

**Review by OrbisK — CHANGES_REQUESTED** [review]
Thank you for you contribution.

Before further review, please add the missing changelog entries. Please see [CONTRIBUTING.md](https://github.com/tusen-ai/naive-ui/blob/main/CONTRIBUTING.zh-CN.md#关于文档和变更日志的格式).

**Comment by XueHua-s** [comment]
Adjusted the wrong place

File: package.json (hunk 1 above)
Comment (XueHua-s) [reply package.json]: The wrong PR has been adjusted

File: CHANGELOG.zh-CN.md [hunk 2]
```diff
@@ -5,6 +5,7 @@
 ### Fixes
 
 - 修复 `n-tree` 的 `override-default-node-click-behavior` 属性可能覆盖掉默认展开按钮和选中按钮的行为
+- 修复 `form-item`'s `validation` input或change后, 再blur，动画触发两次
```
Comment (07akioni) [inline CHANGELOG.zh-CN.md:8]:
```suggestion
- 修复 `form-item`'s `validation` input 或 change 后，再blur，动画触发两次
```

**Comment by 07akioni** [comment]
The issue is actually a bug, which caused by accidentally validation restoration under unexpected behavior. It's fixed in https://github.com/tusen-ai/naive-ui/commit/a604c4bcd886bc13a003d67026ea9b233d1a5ad4

**Pull request closed without merging**